### DISCUSSION TITLE
Custom ports with `--link` how?

### DISCUSSION THREAD

**Discussion by alexandruchircu** [coder/code-server#3811] [Q&A] [answered]
Hi all, I'm trying to test cdr with `--link` and got stuck at step 1. I'm building a custom Docker container (host is Windows 10, behind home router), and running `code-server --link` as entrypoint:

https://github.com/alexandruchircu/testcdr/blob/master/Dockerfile

All is well & I'm given a cdr.co hostname. I then created a bare express server, which listens on port 3000, and this is where the magic doesn't happen; going to 127.0.0.0:3000 on my host gets me the hello world, but going to http://<my_hostname>.cdr.co:3000 gives me a timeout (no response). What am I doing wrong?

Check https://github.com/alexandruchircu/testcdr/blob/master/go.sh if you want to see how I start the container (from WSL).

Thank you!

**Comment by jsjoeio** [comment, marked as answer]
What happens if you go `http://<my_hostname>.cdr.co/proxy/3000`?

**Reply by alexandruchircu** [reply]
Works!
